TypeCobol.Test/TypeCobol.Test.csproj

```diff
@@ -22,7 +22,15 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS</DefineConstants>
```

Comment:
Can you modify these parameters in TypeCobol/Common.props ?
Same remarks for all csproj files.

Reply:
I am not sure that this must be done in this PR because If I see in the develop branch all projects have this CODE_ANALYSIS constant set inside their project's properties.

Reply:
I don't see CODE_ANALYSIS constant in project properties on branch develop.

Reply:
Yes I made a mistake, but it ssems to me that these DEFINEs are project specific.